cawg_identity/src/identity_assertion/assertion.rs

```diff
@@ -63,9 +64,10 @@ impl IdentityAssertion {
     /// Iterator returns a [`Result`] because each assertion may fail to parse.
     ///
     /// Aside from CBOR parsing, no further validation is performed.
-    pub fn from_manifest(
-        manifest: &Manifest,
-    ) -> impl Iterator<Item = Result<Self, c2pa::Error>> + use<'_> {
+    pub fn from_manifest<'a>(
```

Comment:
I am always skeptical when I see lifetimes like this appear in a change.
(Remember our tooling for other languages bindings may not take those that well, and to avoid future changes I would just do the minimal amount of lifetimes specifications possible).

Reply:
Can we preflight this somehow? It will be at best difficult and quite likely impossible to make this change without the new lifetime requirement.